src/controllers/audits.js

```diff
@@ -118,6 +118,23 @@ function AuditsController(dataAccess) {
    * @returns {Promise<Response>} the site's updated audit config
    */
   const patchAuditForSite = async (context) => {
+    function mergeOverrides(existingOverrides, manualOverwrites) {
+      const map = new Map();
+
+      // Add existing overrides to the map
+      existingOverrides.forEach((override) => {
+        map.set(override.brokenTargetURL, override);
+      });
+
+      // Add manual overrides to the map(will overwrite existing if the brokenTargetURL is the same)
+      manualOverwrites.forEach((override) => {
+        map.set(override.brokenTargetURL, override);
+      });
+
+      // Convert the map back to an array
+      return Array.from(map.values());
+    }
+
```

Comment:
this could be simplified and made slightly faster like this:
```suggestion
function mergeOverrides(existingOverrides, manualOverwrites) {
  const overrides = {};

  [...existingOverrides, ...manualOverwrites].forEach(override => {
    overrides[override.brokenTargetURL] = override;
  });

  return Object.values(overrides);
}
```